triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -263,8 +265,15 @@ class Runner(
       case Left(err) => throw new ConverterException(err)
       case Right(x) => x
     })
+    val clientTime: Timestamp =
+      Timestamp.assertFromInstant(Runner.getTimeProvider(timeProviderType).getCurrentTime)
```

Comment:
I am a bit unsure on whether this should depend on the time provider, i.e., be static in static time mode or whether it should always UTC. Given that e.g. DABL always runs in static time mode, I could see it being useful to use UTC since that allows you to feed in time externally.

@tweber-da Given that you raised this issue, do you have a preference here?
or maybe @bame-da?

Reply:
Merging for now. @tweber-da @bame-da let me know if you'd prefer the trigger to get wall clock time in static time mode as well.